rootex/framework/components/visual/model/renderable_component.cpp

```diff
@@ -158,8 +158,8 @@ void RenderableComponent::draw()
 	if (m_LODEnable)
 	{
 		ImGui::Indent();
-		ImGui::DragFloat("LOD Bias", &m_LODBias, 0.01f);
-		ImGui::DragFloat("LOD Distance", &m_LODDistance, 1.0f, 0.0f);
```

Comment:
Instead of `FLT_MAX`, you can use `FLT_MAX / INT_MAX`, as preferred by imgui. Though there's no problem in this too, but why go to the most extreme value :P
Also, you've changed the default value, revert that.

Reply:
Yupp, I tested the speed values, and for LodDistance, the change seems good. But for LODBias, it would be better if we have a precision of 0.01f since even a 0.1 change in its value creates a notable difference, and it becomes a little difficult to use the slider when there are steps of 0.1 itself.

Reply:
Note: https://github.com/ocornut/imgui/blob/master/imgui.h#L523

This recommends using either `FLT_MAX` or `INT_MAX`. I don't think using `FLT_MAX / INT_MAX` makes any sense/ The `/` in the Dear ImGui comment probably means `or` instead of division

Reply:
Oh my bad I got it wrong.